cmd/experimental/kjobctl/pkg/constants/constants.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package constants
+
+const (
+	ProfileLabel = "kjobctl.x-k8s.io/profile"
+	QueueLabel   = "kueue.x-k8s.io/queue-name"
```

Comment:
We should not redefine this.

Reply:
What do you mean?

Reply:
It's defined here https://github.com/kubernetes-sigs/kueue/blob/fd986265f0278a6b1af917ae4850f7d53079e956/pkg/controller/constants/constants.go#L21

We should use it from there.

Reply:
Not sure because we will move it to another repo in future. And to don't relate this repos I created another one.

Reply:
Even if kjobctl is moved the import will be the same, and semantically we want the same thing (set the local queue to be used).

Reply:
Done